src/otelcollector/otelcol-config-extras.yml

```diff
@@ -16,3 +16,35 @@
 #    pipelines:
 #      traces:
 #        exporters: [spanmetrics, otlphttp/example]
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_percentage: 65
+    spike_limit_percentage: 20
+
+  batch:
+    send_batch_max_size: 200
+    send_batch_size: 200
+
+
+  resourcedetection:
+    detectors: [gcp]
+    timeout: 10s
+
+exporters:
+  googlecloud:
+
+  googlemanagedprometheus:
+
+service:
+  pipelines:
+    logs:
+      processors: [batch, memory_limiter, resourcedetection]
```

Comment:
nit: memory_limiter should always come first, if we are using it.  That way, memory usage doesn't continue to accumulate in the batch processor when limits are hit.

Reply:
done